.github/ISSUE_TEMPLATE/02_new-hub_phase-1.yaml

```
@@ -0,0 +1,40 @@
+name: New Hub Turnup Phase 1 - Cloud Account Setup
+description: Track setting up a cloud account for new community on a dedicated cluster
+title: "[New Hub] [Phase 1] Cloud Account Setup - {{community_name}}"
+body:
+  - type: dropdown
+    id: status
+    attributes:
+      label: "What is the 'ready to work' status of this phase?"
+      description: >-
+        This phase can only be marked READY when the table below is complete
+      options:
+        - "NOT READY"
+        - "READY"
+      default: 0
+    validations:
+      required: true
+  - type: textarea
+    id: info-table
+    attributes:
+      label: >-
+        Required information for setting up a cloud account for a new community
+        on a dedicated cluster.
+      description: >-
+        This phase is applicable for cases where a community is being setup on a
+        dedicated cluster. The following table lists the information that must
+        be provided before this phase can start. Use the Notes column to provide
+        any other contextual information.
+      value: |-
+        | Question | Answer | Notes |
+        | :--- | :--- | :--- |
+        | Cloud Provider | `AWS/GCP (delete as appropriate)` | |
+        | Will 2i2c pay for cloud costs? | `Yes/No (delete as appropriate)` | |
+        | Name of cloud account | | |
+        | Community Representative(s) (GitHub handles or email) | | |
+        | Have the Technical Contacts been added to the [AirTable](https://airtable.com/appbjBTRIbgRiElkr/pagD3XyZjqBunYMnC)? | `Yes/No (delete as appropriate)` | |
```

```suggestion
        | Cloud Provider | `AWS/GCP` | |
        | Will 2i2c pay for cloud costs? | `Yes/No` | |
        | Name of cloud account | | |
        | Community Representative(s) (GitHub handles or email) | | |
        | Have the Technical Contacts been added to the [AirTable](https://airtable.com/appbjBTRIbgRiElkr/pagD3XyZjqBunYMnC)? | `Yes/No` | |
```

Too wordy with these